src/core.js

```diff
@@ -4,16 +4,23 @@ import equiv from "./equiv";
 import dump from "./dump";
 import Assert from "./assert";
 import Test, { test, skip, only, pushFailure, generateHash } from "./test";
+import { emit } from "./events";
 import exportQUnit from "./export";
 
 import config from "./core/config";
 import { defined, extend, objectType, is, now } from "./core/utilities";
 import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
+
+import SuiteReport from "./reports/suite";
+
 import { on } from "./events";
 import "./core/onerror";
 
 const QUnit = {};
+const globalSuite = new SuiteReport();
+
+config.currentModule.suiteReport = globalSuite;
```

Comment:
Maybe a comment here should reference the storied "loose unnamed module", because tracking it across files can get confusing.